style.css

```diff
@@ -534,18 +534,30 @@ a:active {
 	display: block;
     text-decoration: none;
     color: white;
+    top: 40px;
+    left: 40px;
 }
 
+.menu {
+    width: 100%;
+}
 .genericon-menu:before {
     color: white;
     font-size: 25px;
 }
 
+.main-navigation.toggled .genericon.genericon-menu:before {
+    content: '\f406';
+}
 .main-navigation.toggled .nav-menu {
-		display: block;
+		z-index: 2;
```

Comment:
To make sure that the menu is on top of everything always. Make this into `z-index: 99999`